checker/jdk/index/src/java/io/Writer.java

```diff
@@ -105,7 +105,7 @@ protected Writer(Object lock) {
      * @throws  IOException
      *          If an I/O error occurs
      */
-    public void write(@NonNegative int c) throws IOException {
+    public void write() throws IOException {
```

Comment:
`write` still takes an `int`, so this should be `public void write(int c) throws IOException {`